crates/iceberg-catalog-bin/src/serve.rs

```diff
@@ -160,3 +171,23 @@ async fn build_nats_client(nat_addr: &Url) -> Result<NatsBackend, Error> {
     };
     Ok(nats_publisher)
 }
+
+fn build_kafka_producer(kafka_brokers: &Vec<SocketAddr>) -> Result<KafkaBackend, Error> {
+    let kafka_brokers_csv = kafka_brokers
+        .iter()
+        .fold(String::new(), |acc, e| format!("{acc},{e}"));
+    let kafka_backend = KafkaBackend {
+        producer: rdkafka::ClientConfig::new()
+            .set("bootstrap.servers", &kafka_brokers_csv)
+            .create()?,
+        topic: CONFIG
+            .kafka_topic
+            .clone()
+            .ok_or(anyhow::anyhow!("Missing kafka topic."))?,
+    };
```

Comment:
Should we accept a `Vec<(SocketAddr, Topic)>`? Are the kafka brokers instances of the same deployment or is it to support separate brokers? If it's separate brokers, then I'd think we'd want flexibility to specify a topic for each of those